specification/security/get_role_mapping/SecurityGetRoleMappingRequest.ts

```diff
@@ -18,15 +18,19 @@
  */
 
 import { RequestBase } from '@_types/Base'
-import { Name } from '@_types/common'
+import { Names } from '@_types/common'
 
 /**
  * @rest_spec_name security.get_role_mapping
  * @since 5.5.0
  * @stability stable
+ * @cluster_privileges manage_security
  */
 export interface Request extends RequestBase {
   path_parts: {
-    name?: Name
+    /**
+     * The distinct name that identifies the role mapping. The name is used solely as an identifier to facilitate interaction via the API; it does not affect the behavior of the mapping in any way. You can specify multiple mapping names as a comma-separated list. If you do not specify this parameter, the API returns information about all role mappings.
+     */
+    name?: Names | null
```

Comment:
I don't think `null` should be here, it will be serialized as `'null'` in the path.
```suggestion
    name?: Names
```

Reply:
Without the test:
```
// Test file: /test/platinum/role_mapping/10_basic.yml
// Test name: Test put role_mapping api
```

is failing with the payload:
```{
  "name": null
}```

Reply:
@delvedor  we need to skip the test then?

Reply:
We can either skip it or update the upstream yaml to not pass the `name` at all.
Very likely the test has been coded in that way because the Java runner doesn't send anything if it sees `null`.

Reply:
I commited the suggestion, let's address this later. The test will be failing for now, but this is a corner case.